src/components/base/Modal/ModalContent.js

```diff
@@ -0,0 +1,44 @@
+// @flow
+
+import React, { PureComponent } from 'react'
+
+class ModalContent extends PureComponent<{
+  children: any,
+  onIsScrollableChange: boolean => void,
+}> {
+  componentDidMount() {
+    this.showHideGradient()
+    if (this._outer) {
+      const ro = new ResizeObserver(this.showHideGradient)
```

Comment:
where is this unlistened? no componentWillUnmount?

Reply:
apparently not with the ResizeObserver API. I guess listener is destroyed with the DOM node